### ISSUE TITLE
Release version 1.0

### ISSUE THREAD

**Issue by adriancole** [openzipkin/zipkin-java#247]
We are nearly a month past the last attempt to release zipkin-java 1.0. At the moment, there are no blocking issues I'm aware of, so I believe we should do it now.

Here's the breakdown of important reasons to do this now, besides getting through blocking issues.

#### Schema compatibility
1.0 has the same schema as zipkin-scala. This is a pro and a con. For example, @mansu wants to revisit the mysql schema and @michaelsembwever wants to improve the cassandra schema. We need to cut a stable version that works with the exact schema before we make any breaking changes. Otherwise, folks using zipkin-scala projects have no bridge. If we never ship 1.0, we can't improve the schema.

#### Model stability
The zipkin model classes are used by other projects like brave and sleuth. These classes have to be guarded or otherwise defended when <1.0 (since we don't have semantic version guarantees). When we release 1.0, it unlocks brave to use zipkin model classes without any repackaging defense.

#### Ecosystem stability
One gate we have to doing a sunset on the old codebase is 1.0 on the new codebase. Until then, we've had double-maintenance, and many questions about what goes where, and unnecessary uncertainty around new features like elasticsearch. For the sake of the community, we need to have closure on this.

In short, lets shipit as we are ready and have lots of real improvements that will be easier once we do.

Releasing 1.0 will allow folks to move to that, and also release pressure that impedes schema

**Comment by adriancole** [comment]
going out now!
